composer.json

```diff
@@ -22,7 +22,24 @@
     },
     "config": {
```

Comment:
* please remove the below line:
`"laminas/laminas-dependency-plugin": true,`
delete the `vendor` directory, delete `composer.lock`, run `composer install` and make sure this plugin is not added to `composer.json`
* not a requirement, rather a nice-to-have, but for consistency across all our packages, let's add under the `config` section:
`"sort-packages": true,`